src/pages/support/faq.md

```diff
@@ -21,7 +21,7 @@ We are adding new events providers regularly, at the time of writing, the follow
 
 * [Platform notifications](https://www.adobe.com/go/devs_events_aep_docs)
 * [Analytics Triggers](https://www.adobe.com/go/devs_events_triggers_docs)
-* [Marketo User Audit Data Stream](../guides/using/marketo-user-audit-data-stream-setup.md)
+* [Marketo User Audit Data Stream](../guides/using/marketo/marketo-user-audit-data-stream-setup.md)
```

Comment:
This should probably link to the base marketo page rather than the marketo audit page